libs/shared/ui/src/lib/styles/components/tag.scss

```diff
@@ -0,0 +1,11 @@
+.tag--normal {
```

Comment:
Not correct, please follow the Figma components

![Capture d’écran 2022-04-25 à 16 38 52](https://user-images.githubusercontent.com/533928/165112075-b91e81da-b36f-4e62-8468-703c8e91421b.png)